### DISCUSSION TITLE
This man deserves all our Achievements for this amazing software

### DISCUSSION THREAD

**Discussion by BigboiKilljoy** [xenesty/AkashaScanner#52] [General]
This scanner is impressive as all hell. i had doubts when i was watching it work on whether or not it was aware of the newest game achievements, until i watched it type in CHENYU VALE and i was like "WOOOOOOO IT KNOWS EVERYTHING". Im currently a broke ass 18yr but you bet your ass when i have money ill be sending some your way, you made my life so easy with this scanner. No way did i want to search through the 1135 achivements i have, NO SIR!

**Comment by VitaliyS** [comment]
Just wanted to say thanks — everything still works perfectly to this day. Absolute lifesaver.
